transforms/__testfixtures__/touchable.output.js

```diff
@@ -2,9 +2,9 @@
 
 <View>
   <TouchableBounce style={{color: 'red'}}>
-    <Image />
-    <Image />
-  </TouchableBounce>
+          <Image />
+          <Image />
+        </TouchableBounce>
```

Comment:
Can do. I wasn't sure if you were okay with a PR that has a failing test (even though it was failing before).